Makefile

```diff
@@ -18,7 +18,10 @@ LOCALE_FILES ?= $(patsubst %,$(OUTPUT_DIR)/locale/%/LC_MESSAGES/app.po,$(LOCALES
 GETTEXT_SOURCES ?= $(shell find $(INPUT_FILES) -name '*.jade' -o -name '*.html' -o -name '*.js' -o -name '*.vue' 2> /dev/null)
 
 # Makefile Targets
-.PHONY: clean makemessages translations
+.PHONY: clean makemessages translations all
+
+all:
+	@echo choose a traget form: clean makemessages translations
```

Comment:
I disagree, I think it's way better if the first and default target is "docs" or "nocommand" instead of "all", so that it's explicitly a command intended just for that, instead of "all" that usually means "build all the things !" 

Also, there's a typo.

Reply:
Hi @vperron,
thanks for commenting on the pull request! :)

Using an "all" target and making it the default is a common pattern.
(For example see https://www.gnu.org/software/make/manual/html_node/Standard-Targets.html#Standard-Targets where is says

> All GNU programs should have the following targets in their Makefiles:
> ‘all’
> Compile the entire program. This should be the default target.
)

The deviations I've seen were if building everything was dangerous somehow or very resource intensive. Both does not seem to be the case here. This is why the default "all" target seems to be natural to me.

Reply:
Alright, why not a default docs target then.

Reply:
@vperron 
Hm, I think I only now understand what your aim was with your first comment. Yes, we could rename the default target to something else, if you prefer. 

Saw that you've already merge the request, thanks!

I did correct two typos in the branch afterwards.

Reply:
Thanks for the corrections :) I was interested in renaming the default target but it was a really minor issue. Please keep on submitting contributions :)